src/main/java/com/example/springbootbookshop/dto/book/CreateBookRequestDto.java

```diff
@@ -1,13 +1,16 @@
 package com.example.springbootbookshop.dto.book;
 
 import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
+import java.util.List;
 
 public record CreateBookRequestDto(@NotBlank String title,
                                    @NotBlank String author,
                                    @NotBlank String isbn,
-                                   @NotBlank @PositiveOrZero BigDecimal price,
+                                   @NotNull @PositiveOrZero BigDecimal price,
                                    String description,
-                                   String coverImage) {
+                                   String coverImage,
+                                   List<Long> categoryIds) {
```

Comment:
you may also validate ids
```suggestion
                                   @NotEmpty List<@Positive Long> categoryIds) {
```